teacher.rb

```diff
@@ -0,0 +1,11 @@
+class Teacher < Person
+  def initialize(age, specialization, name = nil, parent_permission = nil)
```

Comment:
- [ ] parent_permission by default should be `true`.
- [ ] name by default should be `unknown`

`def initialize(age, specialization, name = "Unknown", parent_permission = true)`